test/semanticValidatorTests.ts

```diff
@@ -62,6 +62,14 @@ describe("Semantic validation", () => {
         constants.ErrorCodes.InternalError.name
       );
     });
+    // error path
+    it("should report truly error when error path contains '/providers/Microsoft'", async () => {
+      const specPath = `${testPath}/semanticValidation/specification/validateSwaggerSchema/internalErrors/errorPath.json`;
+      const result = await validate.validateSpec(specPath, undefined);
+      assert(result.validityStatus === false);
+      assert.strictEqual(result.validateSpec?.errors?.[0].code, "ENUM_MISMATCH");
```

Comment:
As this test is to validate the error path, can you add the assert on the error path?